### PR TITLE
docs(README): Add release process to the README file

### PR DESCRIPTION
**Pull request by rafenden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,20 @@ git add .
...
+## Release process
+
+1. Create a new branch based on `master`.
+2. Commit code to the newly created branch following conventions specified by `semantic-release` above.
+3. Create a PR to the `master` branch.
+4. Wait for PR to pass tests and get the required number of approvals.
+5. Once there are no outstanding comments the PR is merged.
+6. Once again tests are being run on `master` branch.
+7. If tests pass, a NPM release is created using version number determined by commit messages.
+8. Storybook is being build and published to `gh-pages` branch which will be available on [GitHub Pages](https://uktrade.github.io/data-hub-components).
```
Comment (yeahfro) [inline README.md]: build 👉  built

Comment (yeahfro) [reply README.md]: > published to `gh-pages`

published to the `gh-pages`

File: README.md [hunk 2]
```diff
@@ -122,6 +122,20 @@ git add .
...
+9. GitHub release is created with commit messages added as changelog.
+10. A new comment is added to the previously merged PR with information about the release.
+11. A tag `released` is added to the PR.
```
Comment (yeahfro) [inline README.md]: `release` is the actual version number? Maybe this could be clearer

**Comment by yeahfro** [comment]
It's probably worth mentioning that the branch is tagged

File: README.md (hunk 2 above)
Comment (rafenden) [reply README.md]: What do you mean?

Comment (yeahfro) [reply README.md]: Apologies, I made a mistake, should have deleted 👍

File: README.md [hunk 3]
```diff
@@ -122,6 +122,20 @@ git add .
...
 
+## Release process
+
+1. Create a new branch based on `master`.
+2. Commit code to the newly created branch following conventions specified by `semantic-release` above.
+3. Create a PR to the `master` branch.
+4. Wait for PR to pass tests and get the required number of approvals.
+5. Once there are no outstanding comments the PR is merged.
+6. Once again tests are being run on `master` branch.
+7. If tests pass, a NPM release is created using version number determined by commit messages.
```
Comment (mcbhenwood) [inline README.md]: Shall we add a link to the NPM package at https://www.npmjs.com/package/data-hub-components ? I found it strangely elusive yesterday.

File: README.md [hunk 4]
```diff
@@ -122,6 +122,21 @@ git add .
...
+## Release process
+
+1. Create a new branch based on `master`.
+2. Commit code to the newly created branch following conventions specified by `semantic-release` above.
+3. Create a PR to the `master` branch.
+4. Wait for PR to pass tests and get the required number of approvals.
+5. Once there are no outstanding comments the PR is merged.
+6. Once again tests are being run on `master` branch.
+7. If tests pass, a [NPM release](https://www.npmjs.com/package/data-hub-components) is created using version number determined by commit messages.
+8. GIT repo is tagged using the same version.
```
Comment (debitan) [inline README.md]: Should this be `The Git` rather than `GIT`?

File: README.md [hunk 5]
```diff
@@ -122,6 +122,21 @@ git add .
...
+9. Storybook is being build and published to the `gh-pages` branch which will be available on [GitHub Pages](https://uktrade.github.io/data-hub-components).
+10. GitHub release is created with commit messages added as changelog.
```
Comment (debitan) [inline README.md]: Maybe 

'A GitHub release is created using the commit messages as the changelog'

Comment (rafenden) [reply README.md]: Thanks, file updated.

File: README.md (hunk 4 above)
Comment (rafenden) [reply README.md]: Thanks, file updated.

File: README.md (hunk 3 above)
Comment (rafenden) [reply README.md]: Good idea, link added.

File: README.md [hunk 6]
```diff
@@ -130,11 +130,12 @@ git cz
...
 5. Once there are no outstanding comments the PR is merged.
 6. Once again tests are being run on `master` branch.
-7. If tests pass, a NPM release is created using version number determined by commit messages.
-8. Storybook is being build and published to `gh-pages` branch which will be available on [GitHub Pages](https://uktrade.github.io/data-hub-components).
-9. GitHub release is created with commit messages added as changelog.
-10. A new comment is added to the previously merged PR with information about the release.
-11. A tag `released` is added to the PR.
+7. If tests pass, a [NPM release](https://www.npmjs.com/package/data-hub-components) is created using version number determined by commit messages.
+8. The Git repo is tagged using the same version.
+9. Storybook is being build and published to the `gh-pages` branch which will be available on [GitHub Pages](https://uktrade.github.io/data-hub-components).
```
Comment (yeahfro) [inline README.md]: `Storybook is *built* and published`

Comment (rafenden) [reply README.md]: Thanks, text updated.

**Pull request merged**